server/guiserver/apps.py

```diff
@@ -108,7 +108,7 @@ def server():
         'jujugui.gzip': options.gzip,
         'jujugui.insecure': options.insecure,
         'jujugui.interactive_login': options.interactivelogin,
-        'jujugui.jem_url': options.jemurl,
+        'jujugui.bundleservice_url': options.bundleservice_url,
```

Comment:
Is this is where we generate the config.js? If so this will need to be `bundleServiceURL`.

And actually the following line chould be `charmstoreURL`